cypress/component/NotePreview.cy.tsx

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable cypress/no-async-tests */
+import { HideButton, PreviewButton } from "../../src/features/CourtCaseList/NotePreviewButton"
+describe("chevron icons check", () => {
```

Comment:
I noticed that in other files we have a component name in the `describe` section. I'm not super fussed personally but I thought that I'd mention this.